pkg/verification/runme.sh

```diff
@@ -48,13 +48,13 @@ dump() {
 
 do_help() {
 cat <<__EOT__
-Usage: docker run [-v <option>] lfedge/verification [-f <fmt>] [-p <platform>] [--accept-license] version|verification_rootfs|verification_live|verification_raw|verification_iso|verification_net
+Usage: docker run [-v <option>] lfedge/eve-verification [-f <fmt>] [-p <platform>] [--accept-license] version|verification_rootfs|verification_live|verification_raw|verification_iso|verification_net
 
 The artifact will be produced on stdout, so don't forget to redirect
 it to a file or use the /out option below.
 
 Example:
-docker run --rm lfedge/verification verification_raw > verification.raw
+docker run --rm lfedge/eve-verification verification_raw > verification.raw
```

Comment:
Do you have a separate docs PR waiting?
If not I'd expand this example to include a release tag i.e., eve-verification:10.0.0 
If yes, then let's make sure that is done in the docs PR.

Reply:
I have a separate docs PR waiting, but expanding this example to include release tags makes sense. So I will have this change.